### PR TITLE
Ports goonchat

### PR DESCRIPTION
**Pull request by spookydonut**
:cl: Spookydonut
add: Added goonchat 
/:cl:

Ported from vgstation

### CODE REVIEW

**Comment by spookydonut** [comment]
Probably going to want to test this.

File: goon/code/datums/browserOutput.dm [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+	if(loaded)
+		return
+
+	loaded = TRUE
+	winset(owner, "browseroutput", "is-disabled=false")
+	for(var/message in messageQueue)
+		to_chat(owner, message)
+
+	messageQueue = null
+	src.sendClientData()
```
Comment (Rohesie) [inline goon/code/datums/browserOutput.dm]: Could you use /tg/'s version? It's pretty similar to /vg/'s, but already cleaned up.

Comment (spookydonut) [reply goon/code/datums/browserOutput.dm]: Their version of what?

Comment (Rohesie) [reply goon/code/datums/browserOutput.dm]: Of goonchat. The same file in their repo would be this: https://github.com/tgstation/tgstation/blob/master/code/modules/goonchat/browserOutput.dm

Comment (spookydonut) [reply goon/code/datums/browserOutput.dm]: At some point in the future sure but this is something i know works with this codebase without any additional work.

Comment (Rohesie) [reply goon/code/datums/browserOutput.dm]: Fair enough.

**Pull request merged**